test/specs/refunds.specs.ts

```diff
@@ -107,99 +110,19 @@ describe("Refunds", () => {
         });
 
         it("should perform long polling", async () => {
-            const recordPendingData = { ...recordData, status: RefundStatus.PENDING };
-
-            fetchMock.getOnce(
-                recordPathMatcher,
-                {
-                    status: 200,
-                    body: recordPendingData,
-                    headers: { "Content-Type": "application/json" },
-                },
-                {
-                    method: HTTPMethod.GET,
-                    name: "pending",
-                }
-            );
-
-            fetchMock.getOnce(
-                recordPathMatcher,
-                {
-                    status: 200,
-                    body: recordData,
-                    headers: { "Content-Type": "application/json" },
-                },
-                {
-                    method: HTTPMethod.GET,
-                    name: "success",
-                }
-            );
-
-            const request = refunds.poll(uuid(), uuid(), uuid());
-            await sandbox.clock.tickAsync(POLLING_INTERVAL);
-            await sandbox.clock.tickAsync(POLLING_INTERVAL);
-
-            await expect(request).to.eventually.eql(recordData);
+            const call = () => refunds.poll(uuid(), uuid(), uuid());
+            await assertPoll(recordPathMatcher, call, sandbox, successItem, pendingItem);
```

Comment:
no need for `async/await`, you can return the promise :)

Reply:
Most of the other tests are using async-await instead of returning the promise 🤔 I feel it would be more consistent in the univapay-node to keep it this way unless we want to engage in a giant test refactoring.

Reply:
Sure, was just thinking that if the test is just one `await` at the end, use `return` rather than `await`, but since this isn't enforced by the CI it's fine either way!